pkg/services/publicdashboards/api/api.go

```diff
@@ -302,15 +302,10 @@ func (api *Api) DeletePublicDashboard(c *contextmodel.ReqContext) response.Respo
 
 // Copied from pkg/api/metrics.go
 func toJsonStreamingResponse(ctx context.Context, features featuremgmt.FeatureToggles, qdr *backend.QueryDataResponse) response.Response {
-	statusWhenError := http.StatusBadRequest
-	if features.IsEnabled(ctx, featuremgmt.FlagDatasourceQueryMultiStatus) {
-		statusWhenError = http.StatusMultiStatus
-	}
-
 	statusCode := http.StatusOK
 	for _, res := range qdr.Responses {
 		if res.Error != nil {
-			statusCode = statusWhenError
+			statusCode = http.StatusBadRequest
```

Comment:
nit: as you only check for the error in the for loop, you can directly break out once you find one
```suggestion
			statusCode = http.StatusBadRequest
			break
```

Reply:
done in https://github.com/grafana/grafana/pull/90191/commits/ca35a1bfe3bb9751cc91d6906bfac8ccd9e36119 :)